README.md

```diff
@@ -1,20 +0,0 @@
-# Intro to Java: Beginner to Advanced
-
-Welcome to the "Intro to Java: Beginner to Advanced" repository! This repo is designed to help me learn Java and summarize what I've learned by storing practice problems.
-
-## About the Repo
-
-This repo is based on the book "Java: The Complete Reference" by Herbert Schildt. I will be using this book to learn Java and summarize the material in this repository. 
-
-## How to Use the Repo
-
-The repo is organized by chapter and section, with each section containing my summaries of the material covered in the book, along with examples and exercises to practice what I've learned. I can simply navigate to the section I want to study and start exploring.
-
-## Contributing
-
-Since this repository is for my personal use, contributions are not necessary. However, if you have any suggestions or feedback, feel free to contact me.
-
-## Acknowledgments
-
-Special thanks to Herbert Schildt for writing "Java: The Complete Reference" and to the open-source community for their contributions to the Java programming language.
-
```

Comment:
You removed this file from the project. So after merging this file will be remove from the repository and you will not find it in the parent dir.  So, resolve it.